src/components/form/Obs.jsx

```diff
@@ -27,6 +28,14 @@ const Obs = (props) => {
           name={`obs|path=${props.path}|concept=${props.concept}`}
           title={props.dropDownTitle}
         />);
+    } else if (props.widget === 'checkbox') {
+      return (
+        <Field
+          component={CheckBox}
+          name={`obs|path=${props.path}|concept=${props.concept}`}
+          options={props.conceptAnswers}
```

Comment:
@jomadoye  how does this checkbox component work?  If there are multiple concept Answers, does it render multiple checkboxes?